docs/3.0-migration-guide.d/remove_MBEDTLS_X509_ALLOW_UNSUPPORTED_CRITICAL_EXTENSION.md

```diff
@@ -0,0 +1,13 @@
+Remove the X509 parser sensitivity control for an unknown critical extension from config.h
+------------------------------------------------------------------------------------------
+
+It affects users who use the `MBEDTLS_X509_ALLOW_UNSUPPORTED_CRITICAL_EXTENSION`
```

Comment:
I think we should provide a bit more context here. Suggested text: "This change does not affect users of the default configuration; it only affect users who enable this option. The X.509 standard says that implementation must reject critical extensions that they don't recognize, and this is is what Mbed TLS does by default. This option allowed to continue parsing those certificates but didn't provide a convenient way to handle those extensions." Then keep the second paragraph.

Reply:
Done